common/config_register.py

```diff
@@ -33,13 +33,9 @@ def definition(io):
                                   init=0,
                                   has_ce=True,
                                   has_reset=has_reset)
-            m.wire(io.I, reg.I)
-            addr_cmp = mantle.EQ(AddressType.N)
-            m.wire(addr_cmp.I0, io.addr)
-            m.wire(addr_cmp.I1, address)
-            m.wire(addr_cmp.O, reg.CE)
+            cmp = mantle.EQ(AddressType.N)
+            m.wire(reg(io.I, ce=cmp(io.addr, address)), io.O)
```

Comment:
I think this should be `CE` instead of `ce`. This actually still works since in our `__call__` wiring syntax implementation, we map `ce` to `CE`, but I believe this was an artifact of an interface change when we renamed the clock input standard names. We should use `CE` since that's the actual port name on the register.

Reply:
Done.